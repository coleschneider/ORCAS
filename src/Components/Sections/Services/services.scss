@import '../../../stylesheets/layout/colors';


/* ===================================================================
 * # services
 *
 * ------------------------------------------------------------------- */
 .s-services {
  padding-top: 15rem;
  padding-bottom: 20rem;
  background-color: $bg-grey;
  text-align: center;
  overflow: hidden;
  position: relative;
}


/* ------------------------------------------------------------------- 
 * ## services list
 * ------------------------------------------------------------------- */
.services {
  margin-top: 7.2rem;
}

.item-feature {
  margin-bottom: .8rem;
}

.item-feature__icon {
  color: #80d3d2;
  font-size: 4.8rem;
}


/* ------------------------------------------------------------------- 
 * responsive:
 * services
 * ------------------------------------------------------------------- */
@media only screen and (max-width:1200px) {
  .s-services {
    padding-top: 18.4rem;
  }

  .services {
    max-width: 900px;
  }

  .services .item-feature:nth-child(2n+1) {
    padding-right: 30px;
  }

  .services .item-feature:nth-child(2n+2) {
    padding-left: 30px;
  }

}

@media only screen and (max-width:1000px) {
  

  .services .item-feature:nth-child(2n+1) {
    padding-right: 20px;
  }

  .services .item-feature:nth-child(2n+2) {
    padding-left: 20px;
  }

}

@media only screen and (max-width:800px) {
  
  .services {
    margin-top: 4.4rem;
  }

  .services .item-feature:nth-child(n) {
    padding-right: 15px;
    padding-left: 15px;
  }

}

@media only screen and (max-width:700px) {
  .services {
    max-width: 500px;
    text-align: center;
  }

  .services .item-feature {
    width: 100%;
    clear: both;
  }

}

@media only screen and (max-width:600px) {
  
  .services .item-feature:nth-child(n) {
    padding-right: 10px;
    padding-left: 10px;
  }

}

@media only screen and (max-width:400px) {
  
  .services .item-feature:nth-child(n) {
    padding-right: 0;
    padding-left: 0;
    margin-bottom: 0;
  }

}
