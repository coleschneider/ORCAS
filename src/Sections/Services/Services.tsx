import * as React from 'react';
import Fade from 'react-reveal/Fade';
import PlaceholderAvatar from '../../images/avatar.png';

export default () => (
  <section id="services" className="s-services target-section">
    <Fade bottom={true} cascade={true}>
      <div className="row section-header has-bottom-sep">
        <div className="col-full">
          <h1 className="display-1">Our Values and Services</h1>
          <p className="lead">
            Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
            assumenda repudiandae molestiae cumque qui. Amet repellat omnis ea.
          </p>
        </div>
      </div>

      <div className="row services block-1-3 block-m-1-2">
        <div className="col-block item-feature">
          <div className="item-feature__icon">
            <i className="icon-heart" />
          </div>
          <div className="item-feature__text">
            <h3 className="item-title">Giving</h3>
            <p>
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
            </p>
          </div>
        </div>

        <div className="col-block item-feature">
          <div className="item-feature__icon">
            <i className="icon-group" />
          </div>
          <div className="item-feature__text">
            <h3 className="item-title">Activities</h3>
            <p>
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
            </p>
          </div>
        </div>

        <div className="col-block item-feature">
          <div className="item-feature__icon">
            <i className="icon-shield" />
          </div>
          <div className="item-feature__text">
            <h3 className="item-title">More Secure</h3>
            <p>
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
            </p>
          </div>
        </div>

        <div className="col-block item-feature">
          <div className="item-feature__icon">
            <i className="icon-building" />
          </div>
          <div className="item-feature__text">
            <h3 className="item-title">Facility</h3>
            <p>
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
            </p>
          </div>
        </div>

        <div className="col-block item-feature">
          <div className="item-feature__icon">
            <i className="icon-chat" />
          </div>
          <div className="item-feature__text">
            <h3 className="item-title">Fundraise</h3>
            <p>
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
            </p>
          </div>
        </div>

        <div className="col-block item-feature">
          <div className="item-feature__icon">
            <i className="icon-wallet" />
          </div>
          <div className="item-feature__text">
            <h3 className="item-title">Donations</h3>
            <p>
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
            </p>
          </div>
        </div>
      </div>
      <div className="meet-wrap">
        <div className="row">
          <div className="col-full meet-header">
            <h2 className="display-2">Meet the team!</h2>
          </div>
        </div>

        <div className="meet__slide">
          <img alt="Author image" src={PlaceholderAvatar} className="meet__avatar" />
          <p>
            Qui ipsam temporibus quisquam velMaiores eos cumque distinctio nam accusantium ipsum. Laudantium quia
            consequatur molestias delectus culpa facere hic dolores aperiam. Accusantium praesentium corpori.
          </p>
          <div className="meet__author">
            <span className="meet__name">A User</span>
            <a href="#0" className="meet__link">
              @member1
            </a>
          </div>
        </div>
        <div className="meet__slide">
          <img alt="Author image" src={PlaceholderAvatar} className="meet__avatar" />
          <p>
            Excepturi nam cupiditate culpa doloremque deleniti repellat. Veniam quos repellat voluptas animi adipisci.
            Nisi eaque consequatur. Quasi voluptas eius distinctio. Atque eos maxime. Qui ipsam temporibus quisquam vel.
          </p>
          <div className="meet__author">
            <span className="meet__name">Team Member 2</span>
            <a href="#0" className="meet__link">
              @member2
            </a>
          </div>
        </div>
        <div className="meet__slide">
          <img alt="Author image" src={PlaceholderAvatar} className="meet__avatar" />
          <p>
            Repellat dignissimos libero. Qui sed at corrupti expedita voluptas odit. Nihil ea quia nesciunt. Ducimus aut
            sed ipsam. Autem eaque officia cum exercitationem sunt voluptatum accusamus. Quasi voluptas eius distinctio.
          </p>
          <div className="meet__author">
            <span className="meet__name">Team Member 3</span>
            <a href="#0" className="meet__link">
              @member3
            </a>
          </div>
        </div>
      </div>
    </Fade>
  </section>
);
