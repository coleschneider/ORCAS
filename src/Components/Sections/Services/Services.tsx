import * as React from 'react';
import Reveal from 'react-reveal/Reveal';
import './services.scss';
import { createSection, ServiceItem } from 'utils/sectionUtils';


const ServicesWrapper = createSection({
  displayTitle: 'Our Values and Services',
  description: `Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
  assumenda repudiandae molestiae cumque qui. Amet repellat omnis ea.`,
  sectionName: 'services'
})

export default () => (
    <ServicesWrapper>
      <div className="row services block-1-3 block-m-1-2">
        <ServiceItem icon="icon-heart" title="Giving">
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
        </ServiceItem>
        <ServiceItem icon="icon-group" title="Activities">
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
        </ServiceItem>
        <ServiceItem icon="icon-shield" title="More Secure">
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
        </ServiceItem>
        <ServiceItem icon="icon-building" title="Facility">
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
        </ServiceItem>
        <ServiceItem icon="icon-chat" title="Fundraise">
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
        </ServiceItem>      
        <ServiceItem icon="icon-wallet" title="Donations">
              Nemo cupiditate ab quibusdam quaerat impedit magni. Earum suscipit ipsum laudantium. Quo delectus est.
              Maiores voluptas ab sit natus veritatis ut. Debitis nulla cumque veritatis. Sunt suscipit voluptas ipsa in
              tempora esse soluta sint.
        </ServiceItem>      
      </div>
    </ServicesWrapper>
);
