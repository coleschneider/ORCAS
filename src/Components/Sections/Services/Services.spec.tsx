import * as React from 'react';
import Services from './Services';

describe('Services', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<Services />);
    expect(wrapper).toMatchSnapshot();
  });
});
