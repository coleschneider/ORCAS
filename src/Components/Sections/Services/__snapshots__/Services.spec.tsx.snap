// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services matches the snapshot 1`] = `
<Component>
  <div
    className="row services block-1-3 block-m-1-2"
  >
    <ServiceItem
      icon="icon-chat"
      title="Donations and Fundraising"
    >
      Reach out through social media, or sign up for the
       
      <Link
        duration={500}
        offset={0}
        smooth={true}
        spy={true}
        to="contact"
      >
        Newsletter!
      </Link>
    </ServiceItem>
    <ServiceItem
      icon="icon-shield"
      title="More Secure"
    >
      ORCAS' wants to provide the community of oxnard a safe and secure facility
    </ServiceItem>
    <ServiceItem
      icon="icon-building"
      title="Building Plans"
    >
      Here are the building plans, and schematics. Probably go into a bit more detail here once its more clear
    </ServiceItem>
    <ServiceItem
      icon="icon-group"
      title="Senior Activities"
    >
      Painting, gaming, Computer Lessons, and other activities!
    </ServiceItem>
  </div>
</Component>
`;
